drivers/npmx/npmx_shell.c

```diff
@@ -819,6 +835,184 @@ static int cmd_ntc_resistance_hot_get(const struct shell *shell, size_t argc, ch
 	return cmd_ntc_resistance_get(shell, argc, argv, npmx_charger_hot_resistance_get);
 }
 
+static uint32_t thermistor_beta_from_ntc_type_get(npmx_adc_ntc_type_t ntc_type)
+{
+	/* Thermistor beta values taken from PS.
+	 * Chapter: CHARGER — Battery charger.
+	 * Table:  Supported thermistor resistors.
+	 */
+	switch (ntc_type) {
+	case NPMX_ADC_NTC_TYPE_10_K:
+		return 3380;
```

Comment:
I believe that beta depends on NTC resistor material. For 10k NTC it may depend on manufacturer and model.
here it's 3380±1%
https://www.tme.eu/Document/9d2eb9f3eda897a378e818dbe183c915/NTCM-10K-B3380.pdf
her it's 3950
https://www.ntcsensors.com/10k_Type_3_Thermistor_Table/

Reply:
Yes, you are right. I will add a command for setting Betha value

Reply:
How about placing it here:
https://github.com/NordicSemiconductor/npmx-zephyr/blob/main/drivers/npmx/Kconfig
and then this value would be used in `npmx_config.h`

Reply:
But then beta would have to be a dynamic config, not static as it is now.